src/processor.js

```diff
@@ -409,7 +409,7 @@ function postprocess(messages, filename) {
 
 export const processor = {
     meta: {
-        name: "eslint-plugin-markdown/markdown",
+        name: "@eslint/markdown/markdown",
```

Comment:
We should update this test:

https://github.com/eslint/eslint-plugin-markdown/blob/923e440094e334d2a0c89c8b9d58bc20ade86373/tests/processor.test.js#L32-L34